### PR TITLE
Flashloan FNs

### PR DESCRIPTION
**Pull request by RCantu92**
* [0x1274b32d4dfacd2703ad032e8bd669a83f012dde9d27ed92e4e7da0387adafe4](): Txn included `swap`ping out of the flashloaned token into the pool's other token, as well as having a new recipient EOA.
* [0x8ee76291c1b46d267431d2a528fa7f3ea7035629500bba4f87a69b88fcaf6e23](https://bscscan.com/tx/0x8ee76291c1b46d267431d2a528fa7f3ea7035629500bba4f87a69b88fcaf6e23): Txn had the PancakeSwap router send txn initiator profit BNB, which our logic handling trace processing didn't properly handle.

Addressing txns in [this issue](https://github.com/forta-network/starter-kits/issues/333).

### CODE REVIEW

**Comment by RCantu92** [comment]
Increased sharding to address [this](https://github.com/forta-network/starter-kits/issues/304)

**Comment by RCantu92** [comment]
Addressing [this](https://github.com/forta-network/starter-kits/issues/424) issue's FN txns. All, except for those where an endpoint with tracing is required, as I still don't have one I can use for non-Ethereum mainnet networks.

**Review by RCantu92 — COMMENTED** [review]
Overall, LGTM. Only more major thing was the possible "hash collision" I brought, but despite it picking up an unintended value, it would fail later on when it is processed.

The rest are more suggestions.

File: flashloan-detector/src/detectors/uniswap-v2-detector.js [hunk 1]
```diff
@@ -1,43 +1,126 @@
...
-      swaps.map(async (swap) => {
-        const { to, data, amount0Out, amount1Out } = swap.args;
-        const { address } = swap;
+    if (!swaps.length) {
+      const calldata = txEvent.transaction.data;
+      let index = 0;
+      let flashloans = [];
+
+      // Continuously search for the selector within the calldata
+      while ((index = calldata.indexOf(swapFunctionSelector, index)) !== -1) {
```
[the remark this answers is not in the document]
Comment (Vxatz) [reply flashloan-detector/src/detectors/uniswap-v2-detector.js:26]: You're right, but yes, I think we're covered with the try/catch in this case

**Review by Vxatz — APPROVED** [review]

**Pull request merged**